recipes/unstructured/setup.patch

```diff
@@ -0,0 +1,151 @@
+From 278946c0c3089749eaf8cc4bebc04ca55f59fdc9 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Sun, 24 Mar 2024 10:11:34 -0500
+Subject: [PATCH] Update setup.py
+
+---
+ setup.py | 113 +++++++++++++++++++++++++++----------------------------
+ 1 file changed, 56 insertions(+), 57 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index caf25c4..24eaaed 100644
```

Comment:
This patch will be a bit hard to maintain. If you can, please report it upstream so we can get this fixed there.